src/io/opendss.jl

```diff
@@ -590,108 +589,174 @@ function dss2tppm_transformer!(tppm_data::Dict, dss_data::Dict, import_all::Bool
-                push!(tppm_data["bus"], starbus)
-
-                bus1, nodes1 = parse_busname(defaults["buses"][1])
-                bus2, nodes2 = parse_busname(defaults["buses"][2])
-                bus3, nodes3 = parse_busname(defaults["buses"][3])
-
-                for (m, (bus, nodes)) in enumerate(zip([bus1, bus2, bus3], [nodes1, nodes2, nodes3]))
-                    transDict = Dict{String,Any}()
-
-                    transDict["f_bus"] = find_bus(bus, tppm_data)
-                    transDict["t_bus"] = starbus["bus_i"]
-
-                    transDict["name"] = "$(defaults["name"]) winding $m"
-
-                    transDict["br_r"] = PMs.MultiConductorMatrix(parse_matrix(diagm(0 => fill(0.2, nconductors)), nodes, nconductors))
-                    transDict["br_x"] = PMs.MultiConductorMatrix(parse_matrix(zeros(nconductors, nconductors), nodes, nconductors))
-
-                    transDict["g_fr"] = PMs.MultiConductorVector(parse_array(0.0, nodes, nconductors))
-                    transDict["g_to"] = PMs.MultiConductorVector(parse_array(0.0, nodes, nconductors))
-                    transDict["b_fr"] = PMs.MultiConductorVector(parse_array(0.0, nodes, nconductors))
-                    transDict["b_to"] = PMs.MultiConductorVector(parse_array(0.0, nodes, nconductors))
-
-                    # CHECK: unit conversion?
-                    transDict["rate_a"] = PMs.MultiConductorVector(parse_array(defaults["normhkva"], nodes, nconductors))
-                    transDict["rate_b"] = PMs.MultiConductorVector(parse_array(defaults["emerghkva"], nodes, nconductors))
-                    transDict["rate_c"] = PMs.MultiConductorVector(parse_array(defaults["emerghkva"], nodes, nconductors))
-
-                    transDict["tap"] = PMs.MultiConductorVector(parse_array(defaults["taps"][m], nodes, nconductors, 1.0))
-                    transDict["shift"] = PMs.MultiConductorVector(parse_array(0.0, nodes, nconductors))
-
-                    transDict["br_status"] = convert(Int, defaults["enabled"])
-
-                    transDict["angmin"] = PMs.MultiConductorVector(parse_array(-60.0, nodes, nconductors, -60.0))
-                    transDict["angmax"] = PMs.MultiConductorVector(parse_array( 60.0, nodes, nconductors,  60.0))
+            nrw = defaults["windings"]
+            prop_suffix_w = ["", ["_$w" for w in 2:nrw]...]
+            if nrw>3
+                # All of the code is compatible with any number of windings,
+                # except for the parsing of the loss model (the pair-wise reactance)
+                error(LOGGER, "For now parsing of xscarray is not supported. At most 3 windings are allowed, not $nrw.")
+            end
 
-                    transDict["transformer"] = true
-                    transDict["switch"] = false
+            transDict = Dict{String,Any}()
+            transDict["name"] = defaults["name"]
+            transDict["source_id"] = "transformer.$(defaults["name"])"
```

Comment:
Missing `active_phases`?

Reply:
Because of the delta-winding, it is not trivial to support 1 and 2 phase transformers. Supporting such transformers will require rethinking the data model.

Instead, if an option other than {busx, busx.1.2.3, busx.0.1.2.3} is specified for a transformer winding, a warning is thrown. The warning states that only three-phase transformers are allowed, and that the bus is considered to be simply busx. A unit test was added to verify that this warning is thrown.

Reply:
Your explanation makes sense, but because `active_phases` is an expected field, like `source_id`, I think you could just populate it with a standard `transDict["active_phases"] = [1, 2, 3]` until 1 or 2 phase transformers are supported and you need to change it.

Reply:
I added it to both the transformers in "trans_comp" en  "trans"